modules/javafx.controls/src/test/java/test/javafx/scene/control/TableCellTest.java

```diff
@@ -366,6 +366,25 @@ public void testCellInUneditableRowIsNotEditable() {
         assertFalse(cell.isEditing());
     }
 
+    @Test
+    public void testRowIsNotNullWhenAutoSizing() {
+        TableColumn<String, String> tableColumn = new TableColumn<>();
+        tableColumn.setCellFactory(col -> new TableCell<>() {
+
+            @Override
+            protected void updateItem(String item, boolean empty) {
+                super.updateItem(item, empty);
+
+                assertNotNull(getTableRow());
+            }
+        });
+        table.getColumns().add(tableColumn);
+
+        StageLoader loader = new StageLoader(table);
+
+        loader.dispose();
```

Comment:
note that the loader isn't disposed if the test fails - that's why there's an instance field (which will be disposed in cleanup), which should be used here

Reply:
done